redis-cache.php

```diff
@@ -26,12 +26,16 @@
 
 define( 'WP_REDIS_VERSION', $meta['Version'] );
 
+require_once WP_REDIS_PLUGIN_PATH . '/includes/class-autoloader.php';
```

Comment:
Is `WP_REDIS_VERSION` still being defined?

Reply:
`WP_REDIS_VERSION` is now defined dynamically to the version definition in the plugin header comment.